OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -410,6 +410,27 @@ public interface IRenderAboveShroudWhenSelected
 		bool SpatiallyPartitionable { get; }
 	}
 
+	public interface ISelection
+	{
+		int Hash { get; }
+		IEnumerable<Actor> Actors { get; }
+
+		void SetNotifySelection(IEnumerable<INotifySelection> worldNotifySelection);
```

Comment:
This can be removed from the interface by changing `Selection` to query `INotifySelection` from `INotifyCreated.Created`.

Reply:
In INotifyCreated.Created(Actor self),

self.World.WorldActor is null

Reply:
`self` *is* the world actor. Omit the `.World.WorldActor` part.